### ISSUE TITLE
Create GET /teams/teamname/members route

### ISSUE THREAD

**Issue by MiguelMachado-dev** [forjadev/gun-organization#12]
![Image](https://user-images.githubusercontent.com/29252011/256685620-1eb8a8b7-60e0-4fee-9fd5-e41184a8aaad.png)

- Create a route to list all member for selected team
- Create tests

**Comment by joaodslourenco** [comment]
Hey @laracarvalho @liliesdeck, how is your availability for us to start working on this task together? I'm available in the evening during the week, around 8~9 PM :)

**Comment by laracarvalho** [comment]
This time is fine by me, although I think we'll need the db schema. But we can start talking abou solutions and mocking some data to advance with the task.

**Comment by joaodslourenco** [comment]
I agree. I was wondering if we could have this meeting at monday at 8 PM. This way we could use the weekend to study Go focused in web development and have a more productive meeting. Personally, I think I would need that hahah. What do you think?

**Comment by laracarvalho** [comment]
I was actually thinking we could have a quick chat to check every topic that might be interesting to read up before we dive into the task. But if you think it's better to have it at Monday I'm okay with that.

Some topics that would be interesting to catch up on, in my opinion:
- basic database joins
- golang and gin framework
- golang's error handling

**Comment by joaodslourenco** [comment]
No problem by me. Just need to confirm with @liliesdeck and we can have this chat today if possible

**Comment by laracarvalho** [comment]
Hey @joaodslourenco @liliesdeck can we confirm our quick chat today 8pm?

**Comment by joaodslourenco** [comment]
Sure! 8pm is fine by me :)

**Comment by liliesdeck** [comment]
8pm is fine for me too, guys! @laracarvalho @joaodslourenco

**Comment by joaodslourenco** [comment]
As a record of our meeting, here are the steps we discussed:

- URL data validation (in order to check if the team name entered is valid and if not, to return a proper message);
- Search for the team ID by the URL param and, with the ID, check all members from that team.

We scheduled our pair programming at wednesday by 8pm at Forja's discord

**Comment by laracarvalho** [comment]
@joaodslourenco @liliesdeck just want to confirm that we'll be working on the feature today 8pm

**Comment by joaodslourenco** [comment]
Yes, I'll be there
